.ci/run_and_validate_e2e_tests.sh

```diff
@@ -5,16 +5,17 @@ ROOT_DIRPATH="$(dirname "${SCRIPT_DIRPATH}")"
 # login to AWS for byzantine images
 aws ecr get-login-password --region "${AWS_DEFAULT_REGION}" | docker login --username AWS --password-stdin 964377072876.dkr.ecr.us-east-1.amazonaws.com
 
-DEFAULT_CONTROLLER_TAG="kurtosistech/ava-e2e-tests_controller"
+DEFAULT_CONTROLLER_TAG="kurtosistech/ava-e2e-tests_controller"   # TODO This is hardcoded in the full_rebuild_and_run.sh script - this should be parameterized!!!!
 DEFAULT_GECKO_IMAGE="964377072876.dkr.ecr.us-east-1.amazonaws.com/gecko:latest"
```

Comment:
@mieubrisse we need to docker pull this in manually, because I think Kurtosis is _not_ picking up the authentication credentials that are being registered with the docker cli  on line 6 of this script.

this is actually a FR for kurtosis: allow for registering credentials for private docker registries within Kurtosis' docker client so that it can automatically pull from private registries

Reply:
@aaronbuchwald  heads up ^ just realized this is a comment on for things on your branch, not on ours